charts/bundler/templates/deployment.yaml

```diff
@@ -68,6 +70,13 @@ spec:
               port: http
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - name: static-bundler-volume
+              mountPath: {{ .Values.bundler.static_data }}
```

Comment:
We can use a hard coded path here (with a user specified glob at the end)

Reply:
Making this path hardcoded means we need to specify it twice.  Is that an issue?

Reply:
Nope, that's fine